client/landing/gutenboarding/utils.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
🙈 I don't want this to spread, but it's pragmatic and practical for now.

Can we add a `FIXME` comment discouraging reuse of this pattern?